REQUIRE

```diff
@@ -7,3 +7,5 @@ IterableTables 0.8.3
 DataValues 0.4.4
 FileIO 1.0.0
 TableShowUtils 0.1.1
+XLSX 0.4.1
+DataFrames 0.14.1
```

Comment:
Can you put this one line into a file ``test/REQUIRE``? Then we don't add a generic dependency, but only one for test runs.